### ISSUE TITLE
thank you so much about your sharing, we install everthing, but how can we use it in godot made us confuse

### ISSUE THREAD

**Issue by xxinxinnn** [you-win/spout-gd#4]
how to use the basic usage code, cause in the terminal said method'load _from_somewhere_idk"isnt cleared in  the code

**Comment by you-win** [comment]
It's an example function that will not do anything. The name is meant to be read in plain English: "Load from somewhere, I don't know".

The developer is meant to provide their own image.

**Comment by xxinxinnn** [comment]
would you mind to share mee an example scrip running on image spout output or video output, thats will be much helpful. cause i am such a new hand in godot.

**Comment by 0x01011970** [comment]
I am also confused about the usage of this extension.

Before investing time in understanding how to compile it (is there really no pre-compiled assets available?) I would have liked to see an example of how it is used in Godot.

Would someone share an example of how this extension can be used to send in real-time at 60FPS the frames generated by a Godot project to a Spout receiver?

Thanks in advance
